tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/StorageSpec.scala

```diff
@@ -419,6 +419,25 @@ abstract class StorageSpec extends BaseIntegrationSpec {
     }
   }
 
+  "Undeprecating a storage" should {
+
+    "allow uploading a file again" in {
+      deltaClient
```

Comment:
You can put those in a for comprehension

Reply:
Done